api/graphite_proxy.go

```diff
@@ -46,5 +50,20 @@ func NewGraphiteProxy(u *url.URL) *httputil.ReverseProxy {
 		resp.Header.Del("content-length")
 		return nil
 	}
+
+	graphiteProxy.ErrorHandler = func(rw http.ResponseWriter, req *http.Request, err error) {
+		if graphiteProxy.ErrorLog != nil {
```

Comment:
it's unfortunate that we have to copy-paste `ReverseProxy.logf()` simply because it's private... :(